docs/plans/562-tenant-write-token/04-fix-increment-3.md

```
@@ -0,0 +1,75 @@
+git status --porcelain        # expect: only "?? docs/plans/562-tenant-write-token/04-fix-increment-3.md"
+```
+
+## Gates (same rows as the first runbook)
+- **G1** `dotnet build Cluckwork.sln --configuration Release --no-restore` → `0 Warning(s)` / `0 Error(s)`.
+- **G2 narrowed**: `dotnet test tests/Cluckwork.Api.IntegrationTests --configuration Release --no-build --filter "FullyQualifiedName~AccountIdConcurrencyTokenModelTests" --logger "console;verbosity=normal"` → `Passed: 2`, `Failed: 0`.
+
+## INCREMENT 9 — two wording fixes (Mechanical)
+
+### 9a. `docs/plans/562-tenant-write-token/03-fix-increment-2.md` — heading level
+Find this exact line:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)